### Hi there ğŸ‘‹

Iâ€™m Vijini Mallawaarachchiâ€”Iâ€™m a Research Associate at the [Flinders Accelerator for Microbiome Exploration](https://fame.flinders.edu.au/), [Flinders University](https://www.flinders.edu.au/).

I enjoy developing algorithms, writing code, reading and blogging.

My current research work is focused on algorithms and computational models related to construction of metagenome-assembled genomes (MAGs). Metagenomics involves the study of various genetic material obtained from the genomes found in microbial communities. A microbial community can contain a large number of organisms. My work focuses on developing methods and tools to investigate the taxonomic structure of a given metagenomic sample.

- ğŸ”­ Iâ€™m currently working on computational methods to resolve viral genomes
- ğŸŒ± Iâ€™m currently learning graph algorithms and flow decomposition techniques
- ğŸ“« How to reach me: viji.mallawaarachchi@gmail.com
- ğŸ  Check out my personal website: https://vijinimallawaarachchi.com
- ğŸ“ Check out my Medium Blog: https://vijini.medium.com/

### Repo Highlights

#### [GraphBin: Refined Binning of Metagenomic Contigs using Assembly Graphs](https://github.com/Vini2/GraphBin)
First automated tool to make use of assembly graphs to refine binning results from existing metagenomic contig-binning tools.

#### [GraphBin2: Refined and Overlapped Binning of Metagenomic Contigs Using Assembly Graphs](https://github.com/Vini2/GraphBin2)
Next generation of GraphBin that refines binning results from existing metagenomic contig-binning tools using assembly graphs. GraphBin2 also assigns contigs to multiple bins if they are shared between different species.

#### [MetaCoAG: Binning Metagenomic Contigs via Composition, Coverage and Assembly Graphs](https://github.com/Vini2/MetaCoAG)
First stand-alone metagenomic contig-binning tool that makes use of assembly graphs apart from composition and coverage information for binning.

#### [Phables: from fragmented assemblies to high-quality bacteriophage genomes](https://github.com/Vini2/phables)
Resolving high-quality bacteriophage genomes from fragmented assemblies of viral metagenomic data using graph algorithms and flow decomposition techniques.

<!--
![Vini2's github stats](https://github-readme-stats.vercel.app/api?username=Vini2&show_icons=true&theme=buefy)

**Vini2/Vini2** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

-->

