### Hi there 👋

I’m Vijini Mallawaarachchi—I’m a Research Associate in Bioinformatics at the [Flinders Accelerator for Microbiome Exploration](https://fame.flinders.edu.au/), [Flinders University](https://www.flinders.edu.au/). I enjoy developing algorithms, writing code, reading and blogging.

My current research work is focused on algorithms and computational models related to the construction of metagenome-assembled genomes (MAGs). Metagenomics involves the study of genetic material obtained from the genomes found in microbial communities. A microbial community can contain a large number of organisms. My work focuses on developing methods and tools to investigate the taxonomic structure of a given metagenomic sample.

- 🔭 I’m currently working on computational methods to resolve viral genomes
- 🌱 I’m currently exploring flow decomposition and graph machine-learning techniques
- 📫 How to reach me: viji.mallawaarachchi@gmail.com
- 🏠 Check out my personal website: https://vijinimallawaarachchi.com
- 📝 Check out my Medium Blog: https://vijini.medium.com/

### Repo Highlights

#### [✨🧬 GraphBin: Refined Binning of Metagenomic Contigs using Assembly Graphs](https://github.com/Vini2/GraphBin)
The first automated tool to make use of assembly graphs to refine binning results from existing metagenomic contig-binning tools.

#### [☯️🧬 GraphBin2: Refined and Overlapped Binning of Metagenomic Contigs Using Assembly Graphs](https://github.com/Vini2/GraphBin2)
Next generation of GraphBin that refines binning results from existing metagenomic contig-binning tools using assembly graphs. GraphBin2 also assigns contigs to multiple bins if they are shared between different species.

#### [🚦🧬 MetaCoAG: Binning Metagenomic Contigs via Composition, Coverage and Assembly Graphs](https://github.com/Vini2/MetaCoAG)
The first stand-alone metagenomic contig-binning tool that makes use of assembly graphs apart from composition and coverage information for binning.

#### [🫧🧬 Phables: from fragmented assemblies to high-quality bacteriophage genomes](https://github.com/Vini2/phables)
Resolving high-quality bacteriophage genomes from fragmented assemblies of viral metagenomic data using graph algorithms and flow decomposition techniques.

<!--
![Vini2's github stats](https://github-readme-stats.vercel.app/api?username=Vini2&show_icons=true&theme=buefy)

**Vini2/Vini2** is a ✨ _special_ ✨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

-->

